src/js/index.js

```diff
@@ -157,11 +158,29 @@ export const hpe = deepFreeze({
       border: {
         radius: '4px',
       },
+      extend: ({ theme }) => `
+        margin-top: ${theme.global.edgeSize.xxsmall};
+      `,
       shadowSize: 'medium',
     },
     elevation: {
+      // Elevation values were derived from this Figma file.
+      // https://www.figma.com/file/eZYR3dtWdb9U90QvJ7p3T9/HPE-Color-Styles?node-id=405%3A25
+      // Naming in Figma file is strong/default/weak vs. Grommet t-shirt sizing.
+      // As defined here, default is currently mapping to medium.
+      light: {
+        xsmall: '0px 1px 2px #0000003D;',
+        small: '0px 2px 4px #0000003D;',
+        medium: '0px 4px 8px #0000003D;',
+        large: '0px 6px 12px #0000003D;',
```

Comment:
One more update actually, latest designs are showing 3 levels of elevation.
Should be:
small: '0px 2px 4px #0000003D;'
medium: '0px 6px 12px #0000003D;'
large: '0px 12px 24px #0000003D;